src/FormattedMessage.jsx

```diff
@@ -24,7 +24,7 @@ const FormattedMessage = (props) => (
         fontSize: 'inherit',
       };
       /* eslint-disable react/no-danger */
-      return <span style={style} dangerouslySetInnerHTML={{ __html: messageString }} />;
+      return <span automation-id="Formatted-Message" aria-label="formatted text" style={style} dangerouslySetInnerHTML={{ __html: messageString }} />;
```

Comment:
The aria-label should be the text itself. The automation id I feel should be the key itself.

Also, these should be the defaults, We should give users the option to override.

Reply:
@MadaraUchiha-314  Okay, got it. Updated to pass in a custom aria label and also gave it a default value. Changed automation-id. Let me know what you think.